src/hottier.rs

```diff
@@ -51,15 +51,13 @@ pub const MIN_STREAM_HOT_TIER_SIZE_BYTES: u64 = 10737418240; // 10 GiB
 const HOT_TIER_SYNC_DURATION: Interval = clokwerk::Interval::Minutes(1);
 pub const INTERNAL_STREAM_HOT_TIER_SIZE_BYTES: u64 = 10485760; //10 MiB
 pub const CURRENT_HOT_TIER_VERSION: &str = "v2";
+
 #[derive(Debug, serde::Deserialize, serde::Serialize)]
 pub struct StreamHotTier {
     pub version: Option<String>,
-    #[serde(rename = "size")]
     pub size: String,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub used_size: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub available_size: Option<String>,
+    pub used_size: String,
```

Comment:
@nikhilsinhaparseable why was this optional previously, we keep unwrapping it everywhere, mostly before setting it to another value, is this redundant?